Hazel/src/Hazel/ImGui/ImGuiLayer.cpp

```diff
@@ -6,15 +6,16 @@
 #include "examples/imgui_impl_opengl3.h"
 
 #include "Hazel/Application.h"
+#include "Hazel\Log.h"
```

Comment:
I suppose that's Visual Studio's auto-completion fault but consider avoiding Windows-specific use of backslashes for paths. `Hazel\Log.h` -> `Hazel/Log.h`